lua/lv-utils/init.lua

```diff
@@ -1,5 +1,61 @@
 local lv_utils = {}
 
+-- recursive Print (structure, limit, indent)
+local function rPrint(s, l, i)
+  l = l or 100 -- default item limit
+  i = i or "." -- indent string
+  if l < 1 then
+    print "ERROR: Item limit reached."
+    return l - 1
+  end
+  local ts = type(s)
+  if ts ~= "table" then
+    if ts == nil then
```

Comment:
`ts == nil` if `s == nil`, in that case you can move this condition (and use s instead of ts) above  `local ts = type(s)`.